### DISCUSSION TITLE
API access error

### DISCUSSION THREAD

**Discussion by kirilldem** [open-webui/open-webui#5761] [General]
# Bug Report

## Installation Method

Docker run

`docker run -d --network=host --gpus=all -v ollama:/root/.ollama -v /home/ollamaAiO:/app/backend/data -e OLLAMA_BASE_URL=http://localhost:11434 -e OLLAMA_HOST=0.0.0.0 -e OLLAMA_ORIGINS="*" --name open-webuiAiO --restart always ghcr.io/open-webui/open-webui:ollama`

## Environment

- **Open WebUI Version:** v0.3.30
- **Ollama v0.3.11

- **Operating System:** Ubuntu 20.04

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [ x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

I should be able to access the /api/generate end point

## Actual Behavior:

Error 404

## Description

**Bug Summary:**
Since I have updated the docker container to the latest version I am for some reason cannot reach the generate api or chat api. Ollama show running and I can also query some api end point. The webUI works as expected. 

## Reproduction Details

**Steps to Reproduce:**

## Logs and Screenshots

[GIN] 2024/09/27 - 13:00:52 | 200 |      62.313µs |       127.0.0.1 | GET      "/api/version"
INFO:     100.97.138.17:51993 - "GET /ollama/api/version HTTP/1.1" 200 OK
INFO:     100.97.138.17:51993 - "GET /api/version/updates HTTP/1.1" 200 OK
[GIN] 2024/09/27 - 13:04:42 | 200 |      67.475µs |   100.97.138.17 | GET      "/"
[GIN] 2024/09/27 - 13:04:54 | 404 |      549.18µs |   100.97.138.17 | POST     "/api/generate"
[GIN] 2024/09/27 - 13:04:55 | 404 |     272.245µs |   100.97.138.17 | POST     "/api/generate"
[GIN] 2024/09/27 - 13:04:56 | 404 |     344.673µs |   100.97.138.17 | POST     "/api/generate"
[GIN] 2024/09/27 - 13:04:57 | 404 |     242.948µs |   100.97.138.17 | POST     "/api/generate"
[GIN] 2024/09/27 - 13:05:04 | 200 |      46.521µs |   100.97.138.17 | GET      "/api/version"
[GIN] 2024/09/27 - 13:05:20 | 200 |     228.581µs |   100.97.138.17 | GET      "/api/ps"

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Unable to reproduce, let us know your findings!
